data/blog/strawberry-jam-steam-deck.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Strawberry Jamming with Tailscale SSH
+authors:
+  - xe
+date: 2023-06-23
+summary: 'Learn how to use Tailscale SSH by installing the Strawberry Jam mod for Celeste on the Steam Deck!'
+tags:
+  - steamDeck
+  - celeste
+  - SSH
+  - sftp
+  - modding
+  - strawberryJam
+images: ['/images/strawberry-jam-steam-deck/hero.png']
```

Comment:
I think for consistency's sake for now we should use the "standard" `social.png` from our Figma. (I know I had suggested the hero image before! I've been thinking about it in the meantime and I think we ought to have more visual continuity in the links because the domain is still a little less familiar.)

I have a plan for the future here and we can talk about it offline if you're interested!

Reply:
Fair enough! I'll upload something like this
![social](https://github.com/tailscale-dev/tailscale-dev/assets/529003/ce3372d1-6393-422e-9ce6-3b4e87f074a4)